src/types/index.d.ts

```diff
@@ -2,6 +2,13 @@
 export {};
 
 declare global {
+  interface Window {
+		darkMode: {
```

Comment:
```suggestion
    darkMode?: {
```

Since `darkMode` is dependent on the app running _inside_ of electron, we should make it optional. Otherwise we'll think it exists when it doesn't (as only electron makes it available).